<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab</class>
 <widget class="QWidget" name="Lab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="l_slider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>351</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="a_slider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>351</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="b_slider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>351</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pAnuluj">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Zerowanie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pOK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_L">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_A">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_B">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
